File: codeforces/69A/go/69a.go
package main

import "fmt"

func main() {
    n := 0
    x := 0
    y := 0
    z := 0

    fmt.Scan(&n)
    for i := 0; i < n; i++ {
        var a, b, c int
        fmt.Scan(&a, &b, &c)
        x += a; y += b; z += c
    }

    if x == 0 && y == 0 && z == 0 {
        fmt.Println("YES")
    } else {
        fmt.Println("NO")
    }
}

File: codeforces/69A/py/69a.py
import sys

n = int(input())
x = 0
y = 0
z = 0
for i in range(n):
    a, b, c = map(int, sys.stdin.readline().split())
    x += a
    y += b
    z += c

if x is 0 and y is 0 and z is 0:
    print("YES")
else:
    print("NO")


